client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -256,8 +256,8 @@ class SiteIconSetting extends Component {
 				<FormLabel className="site-icon-setting__heading">
 					{ translate( 'Site Icon' ) }
 					<InfoPopover position="bottom right">
-						{ translate( 'Your site icon will be shown across WordPress.com, in your' +
-							' visitors\' browser tabs, and as a home screen app icon.' ) }
+						{ translate( 'The Site Icon is used as a browser and app icon for your site.' +
```

Comment:
I'm not sure if core strings are used verbatim if matching our strings, but in case they are, the exact string defines the pixel dimension as a placeholder:

>The Site Icon is used as a browser and app icon for your site. Icons must be square, and at least %s pixels wide and tall.

https://translate.wordpress.com/projects/wpcom/fr/default/?filters%5Bterm%5D=site+icon+is+used&filters%5Buser_login%5D=&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bview%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

Rewritten this would be:

```js
{ translate(
	'The Site Icon is used as a browser and app icon for your site.' +
	'Icons must be square, and at least %s pixels wide and tall.',
	{ args: [ 512 ] }
) }
```